client/scripts/views/components/admin_panel.ts

```diff
@@ -0,0 +1,634 @@
+import m, { Vnode } from 'mithril';
+import $ from 'jquery';
+import { OffchainThread, OffchainTag, CommunityInfo, RolePermission, ChainInfo, ChainNetwork, RoleInfo } from 'models';
+import { Button, Classes, Dialog, Icon, Icons, Tag, TagInput, ListItem, Table, Input, List, TextArea, Switch, Tabs, TabItem, RadioGroup, Form, FormGroup, FormLabel } from 'construct-ui';
+import app from 'state';
+import { sortAdminsAndModsFirst } from 'views/pages/discussions/roles';
+import User from './widgets/user';
+import 'components/admin_panel.scss';
+
+const RoleRow: m.Component<{ roledata?, onRoleUpdate?: Function }> = {
+  view: (vnode) => {
+    return (vnode.attrs.roledata?.length > 0) &&
+      m('RoleData', [
+          vnode.attrs.roledata?.map((role) => {
+            const chainOrCommObj = (app.activeChainId())
+              ? { chain: app.activeChainId() }
+              : { community: app.activeCommunityId() };
+            return m('.role-item', { style: 'display: inline-block; padding-right: 6px;' }, [
```

Comment:
@raykyri are we doing in line style now?

Reply:
I think we should keep styles in CSS wherever possible, so this would be better to not make inline, yeah

Reply:
removed all inline-style